src/AIAD.Api/Startup.cs

```diff
@@ -45,8 +39,8 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             #region JWT
-            var jwtAppSettings = this.Configuration.GetSection("Jwt").Get<JwtAppSettings>();
-            services.AddSingleton<JwtAppSettings>(jwtAppSettings);
+            services.Configure<JwtAppSettings>(Configuration.GetSection("Jwt"));
+            var jwtAppSettings = services.BuildServiceProvider().GetRequiredService<JwtAppSettings>();
             services.AddScoped<IJwtService, JwtService>();
```

Comment:
Not moving this to `AddServicesProjectDependencies()`?

Reply:
I missed this service. I've moved it now.